packages/components/src/standalone-comparison-grid/style.scss

```diff
@@ -0,0 +1,64 @@
+// ==========================================================================
+// Comparison Grid
+// ==========================================================================
+@import "../styles/typography.scss";
+
+.standalone-comparison-grid {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+	gap: 20px;
+	padding: 20px;
+
+	.standalone-comparison-grid__column {
+		display: flex;
+		flex-direction: column;
+		gap: 10px;
+	}
+
+	.standalone-comparison-grid__body {
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+		border-collapse: collapse;
+		font-size: 0.875rem;
+		color: var(--color-text-subtle);
+		table-layout: fixed;
+		border: 1px solid #e0e0e0;
+		border-radius: 4px;
+		background-color: #fff;
+		border-spacing: 0;
+		width: 100%;
```

Comment:
```suggestion

```
It doesn't work well when the parent is a grid and we use a gap attribute.
### Before
The pink area overlaps the content
<img width="1482" alt="image" src="https://github.com/Automattic/wp-calypso/assets/38718/d8ad09d9-4831-4059-a759-1cf92058e904">

### After
Elements following the gap instructions
<img width="1459" alt="image" src="https://github.com/Automattic/wp-calypso/assets/38718/4e3f374c-af3f-4000-91c8-f5670e538594">